src/utils/utils.js

```diff
@@ -6,11 +6,10 @@ export function getDealtCard(G) {
   return dealtCard;
 }
 
-export function getDealtCardsForPlayers(order, dealt) {
-  let cards = Array(order.length).fill("");
+export function getDealtCardsForPlayers(dealt) {
```

Comment:
Short answer is no we need this function but it's not *exactly* the same as returning dealt, it creates an array that replaces the empty elements of `dealt` with an empty string. For example: `[, 'T3', 'T9']` => `['', 'T3', 'T9']` but we don't really need a new array, I'll just change the bit that prints it out

If you want a more detailed explanation about why I changed the way `dealt` works:

Before it would be an array in the order the cards were played: If player 2 started, then player three also played a card it might look something like `['T3', 'T9']`. Now dealt is a sparse array, so in this scenario it would be `[,'T3', 'T9]` by this I mean `dealt[0]` is empty/undefined.

Before, another array was needed (`order`) which in this case would be `['1', '2', '0']` this says that player 2 started, followed by player 3 then finally player 1. When I overhauled how the turnorder is worked out, I got rid of this because it made more sense to just change the position rather than the whole order. 
This was also impossible without using phases because `ctx.playOrder` (the property which actually determines the order of the turns) can only be updated with a new array at the start of a phase.